src/lang/query.pl

```diff
@@ -167,6 +169,41 @@
 	atom_concat('v',Atom,Key),
 	term_keys_variables_1(Xs, Ys).
 
+%% ask(+Statement) is nondet.
+%
+% Same as ask/2 with wildcard scope.
+%
+% @deprecated
+%
+% @param Statement a statement term.
+%
+ask(Statement) :-
+  kb_call(Statement).
+
+%% ask(+Statement,+Scope) is nondet.
+%
+% True if Statement term holds within the requested scope.
+% Scope is a term `[Options,QueryScope]->FactScope` where QueryScope
+% is the scope requested, and FactScope the actual scope
+% of the statement being true.
+% Statement can also be a list of statements.
+%
+% ask/2 is a multifile predicate. Meaning that clauses may be
+% decalared in multiple files.
+% Specifically, declaring a rule using the ask operator `?>`,
+% or the ask-tell operator `?+>` will generate a clause of the ask rule.
+%
+% @deprecated
+%
+% @param Statement a statement term.
+% @param Scope the scope of the statement.
+%
+ask(Statement,[Options,QScope]) :-
+	kb_call(Statement, QScope, _, Options).
+
+ask([X|Xs],QScope->FScope) :-
```

Comment:
This is redundant with above clause and can be deleted.

Reply:
I would assume most people would use ask/1, or ask/2 where second argument is the QScope. or?

Reply:
So only ask(Statement, Options) for ask/2?